twitter-langchain/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@coinbase/twitter-langchain",
+  "version": "0.0.5",
+  "description": "Twitter (X) langchain Toolkit extension of CDP Agentkit",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": ["dist"],
+  "scripts": {
+    "build": "tsc",
+    "lint": "npx --yes eslint -c .eslintrc.json src/**/*.ts",
+    "lint:fix": "npx --yes eslint -c .eslintrc.json src/**/*.ts --fix",
+    "format": "npx --yes prettier -c .prettierrc --write \"**/*.{ts,js,cjs,json,md}\"",
+    "format-check": "npx --yes prettier -c .prettierrc --check \"**/*.{ts,js,cjs,json,md}\"",
+    "check": "tsc --noEmit",
+    "test": "npx jest --no-cache --testMatch='**/*_test.ts'",
+    "test:dry-run": "npm install && npm ci && npm publish --dry-run",
+    "test:e2e": "npx jest --no-cache --testMatch=**/e2e.ts --coverageThreshold '{}'",
+    "test:types": "tsd --files src/tests/types.test-d.ts",
+    "clean": "rm -rf dist/*",
+    "prepack": "tsc",
+    "docs": "npx --yes typedoc --entryPoints ./src --entryPointStrategy expand --exclude ./src/tests/**/*.ts",
+    "docs:serve": "http-server ./docs",
+    "dev": "tsc --watch"
+  },
+  "keywords": [
+    "cdp",
+    "sdk",
+    "agentkit",
+    "ai",
+    "agent",
+    "nodejs",
+    "typescript",
+    "twitter",
+    "langchain"
+  ],
+  "dependencies": {
+    "@coinbase/cdp-agentkit-core": "^0.0.8",
+    "@langchain/core": "0.3.21",
```

Comment:
this should probably match what we have in cdp-agentkit-core package.json (same for Twitter chatbot). perhaps we should update the version in core as well?

Reply:
or alternatively, we can downgrade this to match core, so as to reduce any risk of destabilizing core at the moment

Reply:
yeah, so core was bumped to 0.0.8, and both twitter-langchain and the twitter chatbot use 0.0.8 and langchain/core 0.3.21

Reply:
do you want me to bump to 0.0.9?

Reply:
UPDATE: meant to say that the `@langchain/core` package version should probably match what we have in `cdp-langchain`'s package.json (same for Twitter chatbot). perhaps we should update the version in `cdp-langchain` as well, or downgrade the version here to ensure feature parity between the two.

Reply:
(small nit)

Reply:
ahh roger that